docs/development.md

```diff
@@ -21,7 +21,7 @@ The files at [dev/splunk_scripts](https://github.com/mattermost/mattermost-plugi
 
 ### Building And Deployment
 
-To build your plugin use `make`
+To build your plugin use `make`, you can use `MM_DEBUG=1` as an envvar to generate a debug version of the plugin.
```

Comment:
Probably a good a idea to use `MM_DEBUG` inside of the `Makefile` automatically for the `make debug-*` commands, since there are already commands made for creating debug builds. No need to also have the dev upply the env var for the case of creating debug builds. Funny that the readme mentioned these but they don't exist 🙃

That being said, I'm not blocking on this

Reply:
Yes! I thought of that but didn't want to change the docs, I'll do it as you suggest, a thing is a better solution.

Reply:
I didn't realise it was already building a debug version of the webapp using the envvar

https://github.com/mattermost/mattermost-plugin-splunk/blob/master/Makefile#L89-L95